### ISSUE TITLE
Ability to specify multiple download destinations

### ISSUE THREAD

**Issue by VictorJ76** [igbt6/Packt-Publishing-Free-Learning#91]
Is there a way to download and store in multiple destination ?

**Comment by ad-m** [comment]
What do you mean by "multiple download destination"? You can always run the program repeatedly with various configuration files.

**Comment by VictorJ76** [comment]
Yes but, will not that use the captcha bypass tool twice and provocaque and cause surcharges ?

**Comment by mjenczmyk** [comment]
Downloading does not use any capcha services, so if you don't grab at the same script execution you shouldn't be charged anyway. Is it your only concern?
